nodes/nodes/nodes-nodes-adding-node-iso.adoc

```diff
@@ -11,6 +11,7 @@ This process can be used regardless of how you installed your cluster.
 
 You can add one or more nodes at a time while customizing each node with more complex configurations, such as static network configuration, or you can specify only the MAC address of each node.
 Any required configurations that are not specified during ISO generation are retrieved from the target cluster and applied to the new nodes.
+However, `Machine` or `BareMetalHost` resources are not automatically created after a node has been successfully added to the cluster.
```

Comment:
I added this as a sentence to a paragraph, but if anyone thinks this needs to be more visible (like making it a NOTE), let me know

Reply:
Agree that a note may be more evident